include/mbgl/util/instrumentation.hpp

```diff
@@ -0,0 +1,89 @@
+#pragma once
+
+#include <tracy/Tracy.hpp>
```

Comment:
This header won't be found for bazel builds, can you move it inside the `#if MLN_RENDER_BACKEND_OPENGL` block and additionally guard it with `defined(MLN_USE_TRACY )`?

Reply:
everything is guarded now